src/components/TileGroup/TileGroup.tsx

```diff
@@ -6,15 +6,20 @@ import * as classNames from 'classnames';
 import { TagContainer } from '../TagContainer/TagContainer';
 import { Icon } from '../Icon/Icon';
 import { autobind } from '../../utilities/autobind';
-import { getServerMeasures, sortServersByStatusAndName, filterServerByName, getDiskInformationFromMeasurements } from '../../utilities/server';
+import { getServerMeasures, sortServersByStatusAndName, filterServerByName, getDiskInformationFromMeasurements, filterServerByStatus } from '../../utilities/server';
 
 import './TileGroup.scss';
 
 export class TileGroup extends React.PureComponent<ITileGroupProps, void> {
 
     public render(): JSX.Element {
         const { farm } = this.props;
-        const servers = farm.servers.filter((server) => { return filterServerByName(this.props.filter, server.name); }).sort(sortServersByStatusAndName);
+        let servers;
+        if (this.props.filter.indexOf('status:') !== -1) {
```

Comment:
also, why do this in render and not in componentwillreceiveprops only if the filter or farms differ